packages/fluentui/react-northstar/src/themes/teams/components/Loader/loaderStyles.ts

```diff
@@ -30,8 +30,10 @@ export const loaderStyles: ComponentSlotStylesPrepared<LoaderStylesProps, Loader
   svg: ({ props: p, theme: t, variables: v }: ComponentStyleFunctionParam<LoaderStylesProps, LoaderVariables>) => {
     const outerAnimation: ICSSInJSStyle = {
       animationName: {
-        to: {
-          opacity: 1,
+        keyframe: {
```

Comment:
Just curious does adding this property keep the loader working with fela ?

Reply:
Yed, it does